fix: add cheehwatang

cheehwatang opened the pull request:
<!-- Replace <`issueNumber`> with the Issue number to link it with this PR -->
<!-- Example: #1 links this PR to the first Issue-->

<!-- If you are adding your profile, follow the steps listed here, and delete the **Other additions/changes** section below. -->

resolves #1823 

## Checklist for profile addition

- [ ] Name the PR `fix: add <your-username>`.
- [ ] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [ ] You've linked the PR to the correct issue.

---

## Other additions/changes

<!-- If you are adding a new feature, fixing a bug, or doing any other change, follow the steps listed here, and delete the **Checklist for profile addition** section above. -->

<!-- Tick the checkboxes to ensure you've done everything correctly -->

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

## This PR adds/fixes

<!-- List the changes here and the benefits they bring. -->

## Notes

<!-- List anything note-worthy here (potential issues, PR #302 needs to be merged to before working, etc.). -->

at-the-vr reviewed (approved):
Looks Perfect, one thing - Many profiles also include their Profile Pictures and its considered good if we see that in the [website](https://eddiehubcommunity.github.io/awesome-github-profiles/profiles)

cheehwatang commented:
> Looks Perfect, one thing - Many profiles also include their Profile Pictures and its considered good if we see that in the [website](https://eddiehubcommunity.github.io/awesome-github-profiles/profiles)

Updated with a new preview picture including my profile picture.

I have also updated the preview in the issue accordingly.

Thank you for the comment.

Panquesito7 commented on profiles/cheehwatang.json:
```diff
@@ -0,0 +1,6 @@
+  "issueId": 1823,
+  "name": "Chee Hwa Tang",
+  "username": "cheehwatang"
+}
```
Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

Panquesito7 reviewed (approved):
Thanks! 🎉
